RoboFilePlugin.php

```diff
@@ -0,0 +1,55 @@
+<?php
+/**
+ * This is project's console commands configuration for Robo task runner.
+ *
+ * @see http://robo.li/
+ */
+class RoboFilePlugin extends \Robo\Tasks
+{
+   /**
+    * Minify all
+    *
+    * @return void
+    */
+   public function minify()
+   {
+      $this->minifyCSS();
+      $this->minifyJS();
+   }
+
+   /**
+    * Minify CSS stylesheets
+    *
+    * @return void
+    */
+   public function minifyCSS()
+   {
+      $css_dir = __DIR__ . '/css';
+      if (is_dir($css_dir)) {
+         foreach(glob("$css_dir/*.css") as $css_file) {
+            $this->taskMinify($css_file)
+               ->to("$css_file.min")
+               ->type('css')
+               ->run();
+         }
+      }
```

Comment:
I did some tests on my side, your sub functions should return $this to be chained like this in minify function:
```php
$this->minifyCSS()
     ->minifyJS();
```